### PR TITLE
machine translation feature

### PR DESCRIPTION
**Pull request by nr-kkenney**
## Summary

Machine translation feature branch. With this PR merge, workflows will be updated and can run against the machine translation projects. Files identified for machine translation will be queued, sent, and finally, downloaded with these updates.

Additionally:
* local development has been improved through use of docker-compose
* team & julien have been added as reviewers to all machine translation PRs

### CODE REVIEW

File: scripts/actions/check-job-progress.js [hunk 1]
```diff
@@ -7,8 +7,9 @@ const {
```
Comment (aswanson-nr) [inline scripts/actions/check-job-progress.js:7]: It's probably worth adding a `'use strict;'` to this file, for safety

File: src/components/MachineTranslationCallout.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        <Icon name="globe" size="2em" />
+        <div>
+          <h4>{t('machineTranslation.calloutHeading')}</h4>
+          <Trans i18nKey="machineTranslation.calloutBody" parent="p">
+            In the event of any inconsistency between the English version and
+            the translated version, the{' '}
+            <Link to={englishHref} shouldAutoLocalize={false}>
+              English version
+            </Link>
+            shall prevail. Please visit{' '}
```
Comment (aswanson-nr) [inline src/components/MachineTranslationCallout.js:52]: "shall prevail" conjures images of pointy hats and magic wands for me. Would something like "In the event of any inconsistency between the English version and the translated version, the English version will take priority." This is just my opinion, feel free to disregard.

**Review by aswanson-nr — COMMENTED** [review]
@rudouglas and I reviewed, we had a few comments, but nothing major!

File: scripts/actions/__tests__/add-files-to-translation-queue.test.js [hunk 3]
```diff
@@ -0,0 +1,219 @@
+'use strict';
+const { describe, expect, test, beforeEach } = require('@jest/globals');
```
[the remark this answers is not in the document]
Comment (nr-kkenney) [reply scripts/actions/__tests__/add-files-to-translation-queue.test.js:2]: not needed to run, but added these for intellisense in the ide during development.

[the remark this answers is not in the document]
Comment (aswanson-nr) [reply scripts/actions/__tests__/add-files-to-translation-queue.test.js]: Gotcha! Doing something like this
```
{
  "env": {
    "jest/globals": true
  }
}
``` 
in our eslint config should also work, but no biggy if it doesn't.

**Review by aswanson-nr — APPROVED** [review]

**Pull request merged**